roles/kustomize_deploy/tasks/execute_step.yml

```diff
@@ -125,16 +128,16 @@
            }}
         cifmw_ci_gen_kustomize_values_userdata: >-
           {{
-            (cifmw_architecture_user_kustomize is defined and
-             cifmw_architecture_user_kustomize[_stage_name_id][_name] is defined
+            (_cifmw_kustomize_deploy_user_kustomize is defined and
```

Comment:
`_cifmw_kustomize_deploy_user_kustomize` would always be defined I think. We can probably make those checks smaller then.

Reply:
Yup, should it fallback to `{}` right?